views/_includes/backButton.pug

```diff
@@ -0,0 +1,2 @@
+if prevRoute
+  a(class='back-button' href=prevRoute.path)= '< Go back'
```

Comment:
Not to critique Nick's design but the "less than" symbol looks kind of unfinished.

The GOV.UK back link uses a nice little triangle, but this story isn't really so much about the visual design of this button.

Let's use a unicode arrow in the meantime (←) and wrap it in an aria-hidden span. 

```html
<a><span aria-hidden="true">← </span>Go back</a>
```